litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx

```diff
@@ -244,28 +273,50 @@ const ImageRegistry = () => {
                     <strong> {t('settings.imageRegistry.public')}</strong>
                   </Typography>
                 </div>
-
-                {registry === 'disabled' && isCustomRegistryEnabled === true ? (
+                {!isAvailable ? (
```

Comment:
Can we use && ? @amityt

Reply:
Have already tested with the current changes. @ishangupta-ds